File: backbrace/lambda/python/res.py
import json
from typing import Any


def send_error(status_code: int, message: str) -> dict[Any, Any]:
    return {
        "statusCode": status_code,
        "body": json.dumps({"message": message}),
    }


def send_status(status_code: int) -> dict[Any, Any]:
    return {
        "statusCode": status_code,
    }


def send_json(status_code: int, body: dict[Any, Any]) -> dict[Any, Any]:
    return {
        "statusCode": status_code,
        "body": json.dumps(body),
        "headers": {
            "Content-Type": "application/json",
        },
    }

File: backbrace/lambda/javascript/res.ts
export const send = (statusCode: number, body: any) => {
  /* if local return json body
  if lambda, needs to return stringified json */

  return {
    statusCode,
    body: JSON.stringify(body),
    headers: {
      "Content-Type": "application/json",
    },
  };
};
